README.md

```diff
@@ -351,3 +391,95 @@ $ npm run lint
 ## Github Actions
 
 All pushes to this repo will be checked with `npm test` and `npm lint`.
+
+## Git Workflow
+
+Our branches (in order of stability are):
+
+| Branch     | Environment | AWS Account    | Link To Application                                                                        |
+| :--------- | :---------- | :------------- | :----------------------------------------------------------------------------------------- |
+| main       | development |                | [localhost:3000](http://localhost:3000)                                                    |
+| qa         | qa          | nypl-dams-dev  | [https://qa-new-digitalcollections.nypl.org/](https://qa-new-digitalcollections.nypl.org/) |
+| production | production  | nypl-dams-prod | [https://new-digitalcollections.nypl.org/](https://new-digitalcollections.nypl.org/)       |
+
+## Contributing
+
+1.  Feature branches are cut from `main` using
+
+    `git checkout -b feature/DR-123/feature-name`.
+
+2.  Add your changes to the CHANGELOG.md file under `## [Unreleased]`. See [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) for formatting docs.
+3.  Create a pull request of the `feature` branch _into_ `main`.
+4.  After a feature branch has approved accessibility review and VQA, merge `feature` branch into `main`, then merge `main` branch into `qa` for testing.
```

Comment:
How is `main` merged into `qa`? Is it a pull request or manual merge? Regardless of how, also add that a push to the `qa` branch will trigger a deployment for the qa server.